device/cuda/src/seeding/spacepoint_binning.cu

```diff
@@ -10,7 +10,8 @@
 #include "traccc/cuda/utils/definitions.hpp"
 
 // Project include(s).
-#include "traccc/device/get_prefix_sum.hpp"
+#include "traccc/cuda/utils/make_prefix_sum_buff.hpp"
+#include "traccc/device/fill_prefix_sum.hpp"
```

Comment:
Is it really necessary to include `fill_prefix_sum.hpp` explicitly here? I know that right now that header is included in a **lot** of places. But I thought this file didn't need it explicitly. :thinking: